### PR TITLE
#192 Allow to skip ssl verification

### LINKED ISSUE #192: Provide a feature flag for disabling SSL verification
As a user, I would like to be able to use the Dynatrace API which only has a self-signed certificate.

**Tasks:**
- Introduce an environment variable `HTTP_SSL_VERIFY`, which allows disabling the SSL verification.
- Provide a docu how to disable SSL verification.

### PR DESCRIPTION
**Pull request by agrimmer**
Closes #192

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,7 +74,10 @@ The service is subscribed to the following [Keptn CloudEvents](https://github.co
  You can configure whether these entities should be generated within your Dynatrace tenant by the environment variables specified in the provided [manifest](https://raw.githubusercontent.com/keptn-contrib/dynatrace-service/$VERSION/deploy/service.yaml),
  i.e., using the environment variables `GENERATE_TAGGING_RULES` (default `true`), `GENERATE_PROBLEM_NOTIFICATIONS` (default `true`), `GENERATE_MANAGEMENT_ZONES` (default `true`), `GENERATE_DASHBOARDS` (default `true`), and `GENERATE_METRIC_EVENTS` (default `true`).
  
-  
+* The `dynatrace-service` by default validates the SSL certificate of the Dynatrace API.
+If your Dynatrace API only has a self-signed certificate, you can disable the SSL certificate check
+by the environment variable `HTTP_SSL_VERIFY` (default `true`) specified in the [manifest](https://raw.githubusercontent.com/keptn-contrib/dynatrace-service/$VERSION/deploy/service.yaml).
```
Comment (christian-kreuzberger-dtx) [inline README.md:79]: "by **setting** the environment variable"

**Review by christian-kreuzberger-dtx — CHANGES_REQUESTED** [review]
Functionally this looks fine, please address the changes for the README file

File: README.md (hunk 1 above)
Comment (agrimmer) [reply README.md]: Thanks for the improvement!

**Review by christian-kreuzberger-dtx — APPROVED** [review]
lgtm

**Pull request merged**